management/status_checks.py

```diff
@@ -293,6 +293,8 @@ def run_network_checks(env, output):
 	zen = query_dns(rev_ip4+'.zen.spamhaus.org', 'A', nxdomain=None)
 	if zen is None:
 		output.print_ok("IP address is not blacklisted by zen.spamhaus.org.")
+	elif zen == "[timeout]":
+		output.print_warning("Connection to zen.spamhaus.org timed out. We could not determine whether your server's IP address is blacklisted. Please try again later.")
```

Comment:
Maybe we could add a the website endpoint that we point the users to when the IP/domain turns out to be legitimately blocked (`http://www.spamhaus.org/query/ip/<ipv4>`)